src/components/question.cjsx

```diff
@@ -8,20 +8,20 @@ Answer = React.createClass
   displayName: 'Answer'
   propTypes:
     answer: React.PropTypes.shape(
-      id: React.PropTypes.oneOfType(
+      id: React.PropTypes.oneOfType([
         React.PropTypes.string
```

Comment:
ouch, oh yah, i didn't have anything there to tell coffeescript i meant an array